plugins/flytekit-ray/tests/test_ray.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+import ray
+from flytekitplugins.ray import RayConfig
+
+from flytekit import PythonFunctionTask, task
+from flytekit.configuration import Image, ImageConfig, SerializationSettings
+from flytekit.models.core.resource import ClusterSpec, HeadGroupSpec, RayCluster, WorkerGroupSpec
+
+config = RayConfig(
+    address=None,
+    ray_cluster=RayCluster(
+        name="test_ray",
+        cluster_spec=ClusterSpec(
+            head_group_spec=HeadGroupSpec(image="ray.io/ray:1.10.2"),
+            worker_group_spec=[WorkerGroupSpec(group_name="test_group", replicas=3)],
+        ),
+    ),
+    runtime_env={"pip": ["numpy"]},
```

Comment:
is it possible to run our own image in ray?

Reply:
> yup,but we have to use `rayproject/ray:1.8.0` as our base image

hmm, is there any restriction on which Ray version users can use with Flyte?

Reply:
We will also need to be able to configure Ray head/worker images